packages/api-svc/src/service.ts

```diff
@@ -123,41 +142,63 @@ const kafkaProducerOptions: MLKafkaJsonProducerOptions = {
 
 let globalLogger: ILogger;
 
+// tiger_beetle:
+const USE_TIGERBEETLE = process.env["USE_TIGERBEETLE"] || false;
+const TIGERBEETLE_CLUSTER_ID = process.env["TIGERBEETLE_CLUSTER_ID"] || 0;
+const TIGERBEETLE_CLUSTER_REPLICA_ADDRESSES = process.env["TIGERBEETLE_CLUSTER_REPLICA_ADDRESSES"] || "localhost:9001";
+
+const USE_MONGO_ADAPTER = process.env["USE_MONGO_ADAPTER"] || false;
+
+// environment:
+/*
+	- dev		: default properties
+	- jmeter	: barebone_jmeter_perf
+ */
+const ENV_NAME = process.env["ENV_NAME"] || "dev";
+
 export class Service {
 	static logger: ILogger;
 	static app: Express;
 	static expressServer: Server;
 	static tokenHelper: ITokenHelper;
 	static authorizationClient: IAuthorizationClient;
 	static auditClient: IAuditClient;
+	static configClient: IConfigurationClient;
 	static configRepo: ISettlementConfigRepo;
 	static batchRepo: ISettlementBatchRepo;
-	static participantAccountNotifier: IParticipantAccountNotifier;
 	static batchTransferRepo: ISettlementBatchTransferRepo;
 	static matrixRepo: ISettlementMatrixRequestRepo;
 	static metrics: IMetrics;
 	static messageProducer: IMessageProducer;
 	static startupTimer: NodeJS.Timeout;
+	static abAdapter: IAccountsBalancesAdapter;
+	static configCache: ISettlementConfigCacheRepo;
+	static batchCache: ISettlementBatchCacheRepo;
+	static aggregate: SettlementsAggregate;
```

Comment:
The aggregate is not longer needed here, probably some aggregate only dependencies can also be removed (like the cache)